pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp

```diff
@@ -0,0 +1,72 @@
+(librepcb_package dd9a3433-a992-41f4-b8d1-086095a9b54e
+ (name "KEYSTONE_2468")
+ (description "Battery holder (open) for 2 cells AAA with through-hole pins.")
+ (keywords "keystone,battery,holder,cell,aaa")
+ (author "Jessy")
+ (version "0.1")
+ (created 2019-07-29T20:54:43Z)
+ (deprecated false)
+ (category 1c4e8cee-9c15-4dc4-997c-3bee75a0c1c4)
+ (pad 61856d8f-3a7c-4219-b680-451922719511 (name "+"))
+ (pad d6432d99-3eed-4bdc-997a-f96fd80ba685 (name "-"))
+ (footprint 49510f62-6bc1-4f5c-96e3-22da1b110766
+  (name "default")
+  (description "")
+  (pad d6432d99-3eed-4bdc-997a-f96fd80ba685 (side tht) (shape round)
+   (position -6.35 20.32) (rotation 90.0) (size 3.6 1.8) (drill 1.02)
+  )
+  (pad 61856d8f-3a7c-4219-b680-451922719511 (side tht) (shape round)
+   (position 6.35 20.32) (rotation 90.0) (size 3.6 1.8) (drill 1.02)
+  )
+  (polygon 326a1398-0a86-463f-9c7f-efb87600d382 (layer top_placement)
+   (width 0.2) (fill false) (grab_area true)
+   (vertex (position -12.395 26.59) (angle 0.0))
+   (vertex (position 12.395 26.59) (angle 0.0))
+   (vertex (position 12.395 -26.59) (angle 0.0))
+   (vertex (position -12.395 -26.59) (angle 0.0))
+   (vertex (position -12.395 26.59) (angle 0.0))
+  )
+  (polygon d9eecf73-db75-4682-86b8-e9012ca4cc0f (layer top_documentation)
+   (width 0.2) (fill false) (grab_area true)
+   (vertex (position -12.195 26.39) (angle 0.0))
+   (vertex (position 12.195 26.39) (angle 0.0))
+   (vertex (position 12.195 -26.39) (angle 0.0))
+   (vertex (position -12.195 -26.39) (angle 0.0))
+   (vertex (position -12.195 26.39) (angle 0.0))
+  )
+  (polygon 7325ed13-6916-4463-90d8-38577dc2d2be (layer top_placement)
+   (width 0.5) (fill false) (grab_area false)
+   (vertex (position -7.1 23.75) (angle 0.0))
+   (vertex (position -5.6 23.75) (angle 0.0))
+  )
+  (polygon 04f96c67-82f4-4ab5-874c-3a63d89dfe2a (layer top_placement)
+   (width 0.5) (fill false) (grab_area false)
+   (vertex (position 5.6 23.75) (angle 0.0))
+   (vertex (position 7.1 23.75) (angle 0.0))
+  )
+  (polygon 834ff63e-07bd-4150-bb7e-0d3dc5176b06 (layer top_placement)
+   (width 0.5) (fill false) (grab_area false)
+   (vertex (position 6.35 24.5) (angle 0.0))
+   (vertex (position 6.35 23.0) (angle 0.0))
+  )
+  (circle 345403ba-e446-49ff-851f-8e47619858b6 (layer top_placement)
+   (width 0.4) (fill false) (grab_area false) (diameter 2.8) (position 2.35 14.985)
+  )
+  (circle f6e38727-23d6-46f9-a00d-70c3b40b9da0 (layer top_placement)
+   (width 0.4) (fill false) (grab_area false) (diameter 2.8) (position -2.35 -14.985)
+  )
+  (circle 51f81c6a-8bff-4e25-99a1-5f7b9ad42d96 (layer top_placement)
+   (width 0.4) (fill false) (grab_area false) (diameter 2.8) (position 0.0 0.0)
+  )
```

Comment:
What are the circles for? Should they be holes for mechanical mounting? Or are they just helpers *if* the user want's to add them? But then they probably should be top documentation and not top placement, right?

Reply:
We assume the holes are used to push out the batteries from the other side of the PCB.

I added two variants of the board. One with holes, the other without. In both cases, the hole size is shown on the documentation layer.